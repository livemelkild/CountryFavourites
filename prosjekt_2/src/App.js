import React from 'react';
import Header from "./components/Header";
import Country from "./components/Country";

function App() {
  return (
    <div>
          <Header />

          <Country
          name = "Colombia"
          description = "Teste Colombia"
          />
          <Country
          name = "Norway"
          description = "Teste Norway"
          />
          <Country
          name = "Italy"
          description = "Teste Italy"
          />
          <Country
          name = "Norway"
          description = "Teste Norway"
          />

    </div>
  );
};

export default App;
