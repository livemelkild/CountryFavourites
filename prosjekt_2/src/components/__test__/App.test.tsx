import React from 'react';
import App from '../../App';
import renderer from 'react-test-renderer';
import ThemeContextProvider from '../ThemeContext';

it('renders correctly', () => {
  const tree = renderer
    .create(<ThemeContextProvider><App /></ThemeContextProvider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});