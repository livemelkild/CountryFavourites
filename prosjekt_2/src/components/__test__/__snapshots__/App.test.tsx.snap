// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    className=" "
  >
    <h1>
      Hvilket land vil du reise til?
    </h1>
  </header>
  <div
    className="content_box"
  >
    <div
      id="circle"
    >
      <a
        className="start_link"
        href="/country"
        onClick={[Function]}
      >
        <br />
        <img
          src="/Image/arrow.svg"
        />
        <div
          className="start_side"
        >
          Utforsk landene du kan reise til
        </div>
      </a>
    </div>
  </div>
</div>
`;
